Extensions/Video/JsExtension.js

```diff
@@ -634,7 +634,11 @@ module.exports = {
         this._pixiObject.texture = this._getVideoTexture();
       }
 
-      if (!this._pixiObject.texture.baseTexture.source.paused) {
+      if (
+        this._pixiObject.texture.baseTexture.hasLoaded &&
+        this._pixiObject.texture.baseTexture.source.nodeName == "VIDEO" &&
+        !this._pixiObject.texture.baseTexture.source.paused
+      ) {
         this._pixiObject.texture.baseTexture.source.pause();
```

Comment:
I wonder if there isn't a simpler way to have the video not autoplay... it seems like we're "hacking" pause on top on the video, while there is an autoplay argument in fromVideo

Reply:
`fromVideo `in pixi V3 havent autoPlay argument , [Docs pixi v3](https://pixijs.download/v3.0.11/docs/PIXI.VideoBaseTexture.html)

Reply:
Aaaaah good catch. In this case, let's make a priority to update to Pixi v4 as it's a good reason.
In this case too, let's assume autoplay is prevented and we'll double check this once upgrade to Pixi 4 is done?